### PR TITLE
feat!: upgrade to ucanto@7

### PR DESCRIPTION
**Pull request by Gozala**
Updates to ucanto@7 and removes support for legacy formats. We can release w3up first and then deploy the w3infra to avoid loosing any data

### CODE REVIEW

File: upload-api/test/access-verifier.js [hunk 1]
```diff
@@ -60,8 +62,9 @@ export const create = ({ id }) => {
   const client = Client.connect({
     id,
     channel: server,
-    encoder: CAR,
-    decoder: CBOR,
+    // encoder: CAR,
+    // decoder: CBOR,
```
Comment (Gozala) [inline upload-api/test/access-verifier.js:66]:
```suggestion
```

File: upload-api/test/helpers/resources.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import { S3Client, CreateBucketCommand } from '@aws-sdk/client-s3'
 import { DynamoDBClient, CreateTableCommand } from '@aws-sdk/client-dynamodb'
 import * as Signer from '@ucanto/principal/ed25519'
 import * as Server from '@ucanto/server'
-import { CAR, CBOR } from '@ucanto/transport-legacy'
+import * as Legacy from '@ucanto/transport/legacy'
+// import { CAR, CBOR } from '@ucanto/transport-legacy'
```
Comment (Gozala) [inline upload-api/test/helpers/resources.js:8]:
```suggestion
```

File: upload-api/functions/ucan-invocation-router.js [hunk 3]
```diff
@@ -108,90 +108,41 @@ async function ucanInvocationRouter(request) {
...
-    out.push(receipt.data.out.error || receipt.data.out.ok)
-    forks.push(processInvocationReceipt(receipt.bytes, processingCtx))
-  }
+  // Execute invocations
+  const outgoing = await Server.execute(incoming, server)
+  const response = CAR.response.encode(outgoing)
 
-  await Promise.all(forks)
-  const response = await CBOR.encode(out)
+  processAgentMessageArchive(response, processingCtx)
```
Comment (vasco-santos) [inline upload-api/functions/ucan-invocation-router.js:131]: we were awaiting on this before. Should we keep until we wire up aa dead letter queue?

Comment (Gozala) [reply upload-api/functions/ucan-invocation-router.js:131]: Ah good catch, I didn't intend to change this

```suggestion
  await processAgentMessageArchive(response, processingCtx)
```

**Comment by Gozala** [comment]
Ok finally fixed the segfault, it was caused by aws minor version updates that went into the lockfile after package updates. End up resolving by getting rid of those updates

File: upload-api/functions/ucan-invocation-router.js (hunk 3 above)
Comment (vasco-santos) [reply upload-api/functions/ucan-invocation-router.js:131]: @Gozala this still did not get changed. Maybe commit got lost

**Review by vasco-santos — APPROVED** [review]
LGTM just needs the promise to be awaited in the UCAN handler

**Pull request merged**